### PR TITLE
Mingw: Fixes compiling again

### PR DESCRIPTION
**Pull request by Sonicadvance1**
Getting the CI machines setup to handle this so we can stop breaking it.

### CODE REVIEW

**Review by lioncash — APPROVED** [review]

File: Source/Tools/FEXLoader/WindowsDummyHandlers.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#ifdef _WIN32
+#include "WindowsDummyHandlers.h"
```
Comment (bylaws) [inline Source/Tools/FEXLoader/WindowsDummyHandlers.cpp:2]: Did you miss adding this file?

Comment (Sonicadvance1) [reply Source/Tools/FEXLoader/WindowsDummyHandlers.cpp:2]: Yes I did. I thought this was already there but I had missed it on my first commit of mingw support.

File: Source/Tools/FEXLoader/WindowsDummyHandlers.h [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  }
+
+  FEXCore::HLE::SyscallABI GetSyscallABI(uint64_t Syscall) override {
+    // Don't do anything
+    return {0, false, 0};
+  }
+
+  // These are no-ops implementations of the SyscallHandler API
+  std::shared_mutex StubMutex;
+  FEXCore::HLE::AOTIRCacheEntryLookupResult LookupAOTIRCacheEntry(FEXCore::Core::InternalThreadState *Thread, uint64_t GuestAddr) override {
```
Comment (bylaws) [inline Source/Tools/FEXLoader/WindowsDummyHandlers.h:22]: You can drop this mutex now the LookupAOTIRCacheEntry return type has changed

Comment (Sonicadvance1) [reply Source/Tools/FEXLoader/WindowsDummyHandlers.h:21]: Indeed. Removed.

**Pull request merged**